src/Makefile

```diff
@@ -110,7 +112,7 @@ memento_test_LDFLAGS := ${COMMON_LDFLAGS} -lcurl
 
 include ../build-infra/cpp.mk
 
-ROOT := $(abspath $(shell pwd)/../)
+ROOT := ..
```

Comment:
Why this change?  If you want this to affect the `cpp.mk` snippet doesn't it have to be above the `include`?

Also have you checked that the `alarmdefintions.h` file is being correctly generated with this change?